### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by arunx2**
### Description
Fixed the examples to work with data prepper latest image.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [NA] New functionality includes testing.
- [NA] New functionality has been documented.
  - [NA] New functionality has javadoc added
- [X] Commits are signed with a real name per the DCO

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/data-prepper/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by oeyh — COMMENTED** [review]
Thank you for doing this! Looks good to me, just a few small things:

File: examples/adot/docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
   # AWS OTel Collector
   aws-ot-collector:
     container_name: aws-ot-collector
-    image: otel/opentelemetry-collector:0.24.0
+    image: otel/opentelemetry-collector:0.64.1
     command: ["--config=/etc/aws-ot-collector-config.yml", "--log-level=DEBUG"]
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/adot/docker-compose.yml:7]: Thanks for pointing it out.

File: examples/adot/docker-compose.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ services:
     container_name: data-prepper
     image: opensearchproject/data-prepper:latest
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/adot/docker-compose.yml:33]: Should we limit the opensearch and dashboards versions too, as they were also pointing to `latest`?

File: examples/adot/docker-compose.yml [hunk 3]
```diff
@@ -32,7 +32,7 @@ services:
...
     volumes:
-      - ../trace_analytics_no_ssl.yml:/usr/share/data-prepper/pipelines.yaml
+      - ../trace_analytics_no_ssl_2x.yml:/usr/share/data-prepper/pipelines/pipelines.yaml
       - ../data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/adot/docker-compose.yml]: done

File: examples/jaeger-hotrod/docker-compose.yml [hunk 4]
```diff
@@ -5,7 +5,7 @@ services:
     container_name: data-prepper
     image: opensearchproject/data-prepper:latest
     volumes:
-      - ../trace_analytics_no_ssl.yml:/usr/share/data-prepper/pipelines.yaml
+      - ../trace_analytics_no_ssl_2x.yml:/usr/share/data-prepper/pipelines/pipelines.yaml
       - ../data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/jaeger-hotrod/docker-compose.yml]: done

File: examples/jaeger-hotrod/docker-compose.yml [hunk 5]
```diff
@@ -5,7 +5,7 @@ services:
     container_name: data-prepper
     image: opensearchproject/data-prepper:latest
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/jaeger-hotrod/docker-compose.yml]: done!

File: examples/zipkin-sleuth-webmvc-example/docker-compose.yml [hunk 6]
```diff
@@ -39,7 +39,7 @@ services:
     container_name: data-prepper
     image: opensearchproject/data-prepper:latest
     volumes:
-      - ../trace_analytics_no_ssl.yml:/usr/share/data-prepper/pipelines.yaml
+      - ../trace_analytics_no_ssl_2x.yml:/usr/share/data-prepper/pipelines/pipelines.yaml
       - ../data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/zipkin-sleuth-webmvc-example/docker-compose.yml]: done

File: examples/trace-analytics-sample-app/docker-compose.yml [hunk 7]
```diff
@@ -5,12 +5,10 @@ services:
     container_name: data-prepper
     image: opensearchproject/data-prepper:latest
```
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/trace-analytics-sample-app/docker-compose.yml]: done

File: examples/adot/docker-compose.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (arunx2) [reply examples/adot/docker-compose.yml]: Updated the PR. Thanks, @oeyh!

**Comment by arunx2** [comment]
Something is not right. let me do it again.

**Pull request closed without merging**